ISSUE_TEMPLATE.md

```diff
@@ -31,6 +31,8 @@ to reach Stable.
       - [ ] Initial API review (if API).  Maybe same PR as design doc. **PR-NUMBER**
         -  Any code that changes an API (`/pkg/apis/...`)
         -  **cc @kubernetes/api**
+      - [ ] Identify shepherd (your SIG lead and/or kubernetes-pm@googlegroups.com will be able to help you). **My Shepherd is:** _REPLACE ME_
```

Comment:
Can you provide a sentence or something on what a shepherd is? "The shepherd is responsible for communicating the feature state and if the feature is on-track to make the release goals."?

Also the format like "Brandon Philips <brandon.philips@coreos.com> (@philips)" or something.